Sprint-3/quote-generator/index.html

```diff
@@ -3,13 +3,28 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Quote generator app</title>
+    <link rel="stylesheet" href="style.css" />
+
     <script defer src="quotes.js"></script>
   </head>
   <body>
-    <h1>hello there</h1>
-    <p id="quote"></p>
-    <p id="author"></p>
-    <button type="button" id="new-quote">New quote</button>
+    <main>
+      <h1>hello there</h1>
+
+      <blockquote></blockquote>
+      <p id="quote">Strive not to be a success, but rather to be of value.</p>
+      <p id="author">Albert Einstein</p>
+
+      <div class="action">
+        <button type="button" id="new-quote">New quote</button>
+        <fieldset class="options">
+          <label class="custom-checkbox">
+            <input type="checkbox" id="toggle-timer" />
```

Comment:
The style applied to a checked checkbox (slight change in color) is not very noticeable.

Reply:
Thank you for pointing that out! I’ve updated the style for the checked checkbox to make the color change more noticeable. 
`.custom-checkbox input[type="checkbox"]:checked {
  background-color: #ec09fc;
  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.4);
}`